### PR TITLE
Migrate code to use updated lib functions

### PR DESCRIPTION
**Pull request by rdhananjaya**
## Purpose
$title

## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

**Review by ldclakmal — APPROVED** [review]

File: learn/writing-secure-ballerina-code.md [hunk 1]
```diff
@@ -76,60 +76,51 @@ Security-sensitive functions and remote methods of Ballerina standard libraries
...
 
-When the Ballerina compiler can determine that a function is returning tainted data without tainted data being passed in as parameters to that function, it is required to annotate the function's return type as `@tainted`. If not, the function author has to clean up the data before returning. For instance, if you are to read from the database and return that result, you either need to annotate that function's return type as `@tainted` or you have to clean up and make sure the returned data is not tainted.
+When the Ballerina compiler can determine that a function is returning tainted data without tainted data being passed in as parameters to that function, it is required to annotate the function's return type `@tainted`. If not, the function author has to clean up the data before returning. For instance, if you are to read from the database and return the result, you either need to annotate that function's return type as `@tainted` or you have to clean up and make sure the returned data is not tainted.
```
Comment (praneesha) [inline learn/writing-secure-ballerina-code.md:123]:
```suggestion
When the Ballerina compiler can determine that a function is returning tainted data without tainted data being passed in as parameters to that function, it is required to annotate the function's return type as `@tainted`. If not, the function author has to clean up the data before returning. For instance, if you are to read from the database and return the result, you either need to annotate that function's return type as `@tainted` or you have to clean up and make sure the returned data is not tainted.
```

File: learn/writing-secure-ballerina-code.md [hunk 2]
```diff
@@ -76,60 +76,51 @@ Security-sensitive functions and remote methods of Ballerina standard libraries
...
-jdbc:Parameter paramId = {sqlType:jdbc:TYPE_VARCHAR, value:studentId};
-var dt = testDB->select("SELECT NAME FROM STUDENT WHERE ID = ?", ResultStudent,
-                        paramId);
+stream<record{}, error> resultStream = jdbcClient->query(`SELECT NAME FROM STUDENT WHERE ID = ${<@untainted> studentId}`);
 ```
 
 Command-line arguments passed to Ballerina programs and inputs received through service resources are considered as tainted. Additionally, return values of certain functions are marked with the `@tainted` annotation to denote that the resulting value should be considered as untrusted data.
 
-For example, the `select` remote method of the `java:jdbc` client highlighted above returns a `@tainted table<record {}>|Error`. This means that any value read from a database is considered as untrusted.
+For example, the `query` remote method of the `java.jdbc` client highlighted above returns a `@tainted stream <record {}, sql:Error>`. This means that any value read from a database is considered as untrusted.
```
Comment (praneesha) [inline learn/writing-secure-ballerina-code.md:121]:
```suggestion
For example, the `query` remote method of the `java.jdbc` client highlighted above returns an `@tainted stream <record {}, sql:Error>` error. This means that any value read from a database is considered as untrusted.
```

Comment (rdhananjaya) [reply learn/writing-secure-ballerina-code.md:121]: `stream <record {}, sql:Error>` this is a stream type, this is not an error

File: learn/writing-secure-ballerina-code.md (hunk 1 above)
Comment (rdhananjaya) [reply learn/writing-secure-ballerina-code.md:123]: @praneesha would it be ok to use `with` instead of `as` here?

Comment (praneesha) [reply learn/writing-secure-ballerina-code.md:123]: +1

File: learn/writing-secure-ballerina-code.md (hunk 2 above)
Comment (praneesha) [reply learn/writing-secure-ballerina-code.md:121]: +1 to say "stream type" instead of "error".

**Review by praneesha — APPROVED** [review]

**Pull request merged**